calibrate.py

```diff
@@ -58,13 +92,29 @@ def write_header_file(board_number, calibration):
     os.rename(fname+'.tmp', fname)
     print('calibration data writed to {}'.format(fname))
 
+
+def write_pickled_data(board_number, calibration):
+    fname = './board_calibration_data/pickled_board_{}'.format(board_number)
+    fpickle = open(fname, 'wb')
```

Comment:
Use the 'with' statement when opening files, as in line 88. That handles closing the file gracefully.